src/defaults/html.pipe.js

```diff
@@ -58,6 +59,7 @@ const htmlpipe = (cont, payload, action) => {
     .before(html).expose('html')
     .before(responsive)
     .once(cont)
+    .after(sanitize)
```

Comment:
this is a wrong place. at this moment, the HTML is already written to the response.body. the `cont` is the step that executes the template engine.

Reply:
forget about the comment above. I thought that the purify would operate on the VDOM and not on the html itself. so in fact, it **is** reparsing the entire output...

Reply:
so I'm no longer sure that this is correct. I would rather have just sanitized the VDOM before it is used by HTL. otherwise you remove too much power from the templating....

Reply:
👍 I see your point. DOMPurify should be able to work on the VDOM, so I'll refactor accordingly if there are no objections from the others

Reply:
But if the sanitization takes place before the `cont`, wouldn't we run the risk of developers introducing potential security issues? I think that parsing the final output would be the safer bet...

Reply:
> wouldn't we run the risk of developers introducing potential security issues? 

that's why they use HTL to minimize this :-) but I think they should still have the power to write things like `<a href="#" onclick="..."/>` , if they want.

I think we should have more examples, of what should be possible and what to guard against.